.drone/scripts/mentors.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+cp files/dput.cf $HOME/.dput.cf
+
+export NEEDED_VERSION="$(cat .data.json | jq -r '.current_pkgver')"
+dput mentors "../${pkgname}_${NEEDED_VERSION}-1_source.changes"
```

Comment:
Do we need to add the GPG key to the system anywhere here, or is that not needed?